src/emm/services/git_service.py

```diff
@@ -135,6 +135,17 @@ def merge_base(self, commit_a: str, commit_b: str, directory: Optional[Path] = N
         with local.cwd(self._determine_directory(directory)):
             return self.git[args]().strip()
 
+    def commit_all(self, commit: str, directory: Optional[Path] = None) -> None:
+        """
+        Get the commit hash of the current HEAD of a repository.
+
+        :param commit: Commit message for all the changes.
+        :param directory: Path to repository to query.
+        """
+        args = ["commit", "-am", commit]
```

Comment:
Can we use long options here (`--all` and `--message`) instead of `-am` so it is more obvious what is happening?